mkdocs/docs/getting-started/install.md

```diff
@@ -56,7 +56,7 @@ curl -s https://raw.githubusercontent.com/iter8-tools/iter8-install/main/install
     set -e
```

Comment:
@kalantar If you agree, I am in favor of removing the ??? info "Look inside instal-prom-add-on.sh" tooltip altogether in install.md. It is more work for us in terms of ensuring consistency/correctness of the documentation. Of course, folks can navigate to the GitHub page and look inside!

Reply:
agree. done.